Sming/third-party/.patches/esp-open-lwip.patch

```diff
@@ -60,6 +60,7 @@ index 1bc584f..0bfc424 100644
 +lwip/app/espconn_tcp.o \
 +lwip/app/espconn_udp.o \
 +lwip/app/espconn_mdns.o \
++lwip/api/err.o \
```

Comment:
When compiling with ENABLE_ESPCONN esconn code uses a function defined in err.c to map an int error code to a string. Absence of err.o would generate linker symbol not found err